source/extensions/filters/network/dubbo_proxy/config.cc

```diff
@@ -2,7 +2,12 @@
 
```

Comment:
This file has slightly low coverage:
https://195194-65214191-gh.circle-artifacts.com/0/coverage/coverage.source_extensions_filters_network_dubbo_proxy_config.cc.html

can you add test for failure case? If they are validated via PGV, then just ASSERT is enough.

Reply:
yes, ASSERT is enough, i have modified it.